ocrd_zip.md

```diff
@@ -133,6 +133,11 @@ Invalid `mets:FLocat/@xlink:href` in `/tmp/foo/ws1/data/mets.xml`:
 All files except `mets.xml` itself that are contained in `data` directory must
 be referenced in a `mets:file/mets:Flocat` in the `mets.xml`.
 
+### When in METS and not in data
```

Comment:
What if those files do not actually exist (like the local `file:/...` links in Goobi/Kitodo output? Or they might not be accessible to checksum?

Reply:
METS files that are compatible with the DFG-Viewer ``MUST`` contain a ``<mets:fileGrp USE="DEFAULT">`` that ``MUST`` be resolvable via HTTP so this should be the last resort as a fallback.

Reply:
The problem is that if we enforce the rule "mets:file in METS -> must be either packaged as payload or listed in fetch.txt" can conflict with the rule "every file listed in fetch.txt must have a checksum and be resolveable". Because references to local files, invalid URLs and non-resolveable URLS are not uncommon in existing METS documents.

Reply:
So I assume we can only relax the latter rule "every file listed in fetch.txt must have a checksum and be resolveable"?

Reply:
No, that rule is part of BagIt https://tools.ietf.org/html/rfc8493#section-2.2.3